src/apps/competitions/tests/test_submissions.py

```diff
@@ -11,6 +11,8 @@
 from api.serializers.competitions import CompetitionSerializer
 from competitions.models import Submission
 from competitions.tasks import run_submission
+
+from apps.competitions.tasks import submission_status_cleanup
```

Comment:
Can this be one line?
```suggestion
from competitions.tasks import run_submission, submission_status_cleanup
```